<?xml version="1.0" encoding="UTF-8" ?>
<runHistory script="NewFunctions">
 <run>
  <RunDate>2024-08-26 15:36:04.344 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.046000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_153604.344/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:37:04.642 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>10.151000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_153704.642/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:39:34.998 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>10.262000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_153934.998/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:40:40.782 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>16.635000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_154040.782/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:41:15.325 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>44.523000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_154115.325/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:42:16.766 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>5.704000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_154216.766/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:43:32.617 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>64.323000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_154332.617/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:51:42.163 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>12.387000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_155142.163/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:54:21.249 -0400</RunDate>
  <Status>Success</Status>
  <Duration>0.067000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20240826_155421.249/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:55:59.619 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>4.173000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20240826_155559.619/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:57:50.770 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.064000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20240826_155750.770/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:58:53.993 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.067000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240826_155853.993/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:59:36.015 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.008000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_155936.015/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>SenseTalk Compiler Exception -- in newfunctions:
Syntax Error at line 23: can&#x27;t understand &quot;record&quot; at character 11</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 15:59:51.321 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.008000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_155951.321/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>SenseTalk Compiler Exception -- in newfunctions:
Syntax Error at line 23: can&#x27;t understand &quot;fourth&quot; at character 11</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:00:05.679 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.007000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_160005.679/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>SenseTalk Compiler Exception -- in newfunctions:
Syntax Error at line 23: can&#x27;t understand &quot;4ourth&quot; at character 9 (an identifier can&#x27;t start with a digit, and &quot;ourth&quot; is not a unit name)</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:01:16.296 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.009000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_160116.296/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>SenseTalk Compiler Exception -- in newfunctions:
Syntax Error at line 23: can&#x27;t understand &quot;record&quot; at character 16</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:01:28.095 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.072000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240826_160128.095/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:01:39.081 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.009000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20240826_160139.081/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>SenseTalk Compiler Exception -- in newfunctions:
Syntax Error at line 23: can&#x27;t understand &quot;record&quot; at character 15</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:03:06.739 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.059000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240826_160306.739/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:08:49.245 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>23.231000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240826_160849.245/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-26 16:12:11.861 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>49.519000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240826_161211.861/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:16:17.261 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.105000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_131617.261/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>WorkbookWriteError -- Error (can&#x27;t open file for writing) saving workbook to file C:/Users/Dave/Documents/TSU_Excel.suite/Resources/Test1.xlsx</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:16:31.794 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.123000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_131631.794/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:17:16.173 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.112000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_131716.173/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:17:27.786 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>42.901000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_131727.786/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:22:40.028 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>24.859000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_132240.028/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:23:22.220 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>14.108000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_132322.220/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:27:02.936 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.075000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_132702.936/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>WorkbookWriteError -- Error (can&#x27;t open file for writing) saving workbook to file C:/Users/Dave/Documents/TSU_Excel.suite/Resources/Test1.xlsx</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:27:11.358 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.118000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_132711.358/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:27:40.560 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.115000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_132740.560/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:28:53.924 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.131000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_132853.924/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:35:24.787 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>10.283000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_133524.787/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:36:21.926 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>120.887000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_133621.926/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:38:30.510 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>119.552000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_133830.510/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:44:35.205 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>26.778000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20240828_134435.205/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STDatabaseError -- Where clause specifies invalid column name &#x27;RecNumber&#x27; (no such column)</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:45:44.484 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.097000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_134544.484/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STDatabaseError -- Where clause specifies invalid column name &#x27;RecNumber&#x27; (no such column)</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:45:59.861 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.073000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_134559.861/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STDatabaseError -- Where clause specifies invalid column name &#x27;RecNumber&#x27; (no such column)</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:46:30.593 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.077000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_134630.593/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STDatabaseError -- Where clause specifies invalid column name &#x27;RecNumber&#x27; (no such column)</ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:46:48.646 -0400</RunDate>
  <Status>Aborted</Status>
  <Duration>75.514000</Duration>
  <Errors>2</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_134648.646/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2024-08-28 13:50:24.867 -0400</RunDate>
  <Status>Failure</Status>
  <Duration>0.105000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>2</Successes>
  <LogFile>20240828_135024.867/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Script Error -- Number of Errors logged: 1</ErrorMessage>
 </run>
</runHistory>
